### PR TITLE
Validate that global configuration does exists

### PR DESCRIPTION
**Pull request by killua99**
New wp cli api to validate that a global configuration does exists.

Related #4954

### CODE REVIEW

File: features/cli.feature [hunk 1]
```diff
@@ -39,3 +39,42 @@ Feature: `wp cli` tasks
...
+           * <config>
+           * : Config name to validate.
+           *
+           * @when after_wp_load
+           */
+          public function __invoke( $args ) {
+              if ( WP_CLI::has_config( $args[0] ) ) {
+                  WP_CLI::log( 'Global configuration `url` does exists.' );
+              } else {
+                  WP_CLI::log( 'Global configuration `dummy` does not exists.' );
```
Comment (schlessera) [inline features/cli.feature]:
```suggestion
                  WP_CLI::log( "Global configuration '{$args[0]}' does not exist." );
```

File: features/cli.feature [hunk 2]
```diff
@@ -39,3 +39,42 @@ Feature: `wp cli` tasks
...
+    When I run `wp --require=custom-cmd.php custom-command url`
+    Then STDOUT should be:
+      """
+      Global configuration `url` does exists.
+      """
+
+    When I run `wp --require=custom-cmd.php custom-command dummy`
+    Then STDOUT should be:
+      """
+      Global configuration `dummy` does not exists.
```
Comment (schlessera) [inline features/cli.feature]:
```suggestion
      Global configuration 'dummy' does not exist.
```

File: features/cli.feature [hunk 3]
```diff
@@ -39,3 +39,42 @@ Feature: `wp cli` tasks
       """
     And STDERR should be empty
     And the return code should be 0
+
+  Scenario: Ability to detect a global configuration does exists or not
```
Comment (schlessera) [inline features/cli.feature]:
```suggestion
  Scenario: Checking whether a global configuration parameter exists or not
```

File: features/cli.feature [hunk 4]
```diff
@@ -39,3 +39,42 @@ Feature: `wp cli` tasks
...
+    Given a WP installation
+    And a custom-cmd.php file:
+      """
+      <?php
+      class Custom_Command extends WP_CLI_Command {
+
+          /**
+           * Custom Command to validate a global configuration does exists or not.
```
Comment (schlessera) [inline features/cli.feature]:
```suggestion
           * Custom command to validate a global configuration does exist or not.
```

File: features/cli.feature [hunk 5]
```diff
@@ -39,3 +39,42 @@ Feature: `wp cli` tasks
...
+           *
+           * <config>
+           * : Config name to validate.
```
Comment (schlessera) [inline features/cli.feature]:
```suggestion
           * : Configuration parameter name to check for.
```

**Comment by schlessera** [comment]
@killua99 The approach looks good. I added a few change requests, which are mostly about grammar only.

**Comment by killua99** [comment]
@schlessera Double check everything seems ok

**Pull request merged**